executor/src/main/resources/application.properties

```diff
@@ -17,4 +17,7 @@ kafka.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule r
 mp.messaging.incoming.events-in.connector=smallrye-kafka
 mp.messaging.incoming.events-in.topic=events
 mp.messaging.incoming.events-in.value.serializer=org.apache.kafka.common.serialization.StringSerializer
-mp.messaging.incoming.events-in.auto.offset.reset=earliest
\ No newline at end of file
+mp.messaging.incoming.events-in.auto.offset.reset=earliest
+mp.messaging.incoming.events-in.group.id=${KAFKA_GROUP_ID:default}
```

Comment:
We do configure the group id for every executor (we use the processorId), so that every executor will receive every event.